src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIWaveEquationSEM.cpp

```diff
@@ -0,0 +1,1027 @@
+  m_receiverConstants.resize( numReceiversGlobal, numNodesPerElem );
+
+}
+
+
+void AcousticVTIWaveEquationSEM::registerDataOnMesh( Group & meshBodies )
+{
+
+  forDiscretizationOnMeshTargets( meshBodies, [&] ( string const &,
+                                                    MeshLevel & mesh,
+                                                    arrayView1d< string const > const & )
+  {
+    NodeManager & nodeManager = mesh.getNodeManager();
+
+    nodeManager.registerField< fields::Pressure_p_nm1,
+                               fields::Pressure_p_n,
+                               fields::Pressure_p_np1,
+                               fields::Pressure_q_nm1,
+                               fields::Pressure_q_n,
+                               fields::Pressure_q_np1,
+                               fields::PressureDoubleDerivative,
+                               fields::ForcingRHS,
+                               fields::MassVector,
+                               fields::DampingVector_p,
+                               fields::DampingVector_pq,
+                               fields::DampingVector_q,
+                               fields::DampingVector_qp,
+                               fields::StiffnessVector_p,
+                               fields::StiffnessVector_q,
+                               fields::StiffnessVector, //Debug
+                               fields::DampingVector, //Debug
+                               fields::FreeSurfaceNodeIndicator,
+                               fields::LateralSurfaceNodeIndicator,
+                               fields::BottomSurfaceNodeIndicator >( this->getName() );
+
+
+    FaceManager & faceManager = mesh.getFaceManager();
+    faceManager.registerField< fields::FreeSurfaceFaceIndicator >( this->getName() );
+    faceManager.registerField< fields::LateralSurfaceFaceIndicator >( this->getName() );
+    faceManager.registerField< fields::BottomSurfaceFaceIndicator >( this->getName() );
+
+    ElementRegionManager & elemManager = mesh.getElemManager();
+
+    elemManager.forElementSubRegions< CellElementSubRegion >( [&]( CellElementSubRegion & subRegion )
+    {
+      subRegion.registerField< fields::Delta >( this->getName() );
+      subRegion.registerField< fields::Epsilon >( this->getName() );
+      subRegion.registerField< fields::F >( this->getName() );
+      subRegion.registerField< fields::MediumVelocity >( this->getName() );
+      subRegion.registerField< fields::PartialGradient >( this->getName() );
+    } );
+
+    arrayView1d< real32 > const p_dt2 = nodeManager.getField< fields::PressureDoubleDerivative >();
+    int const rank = MpiWrapper::commRank( MPI_COMM_GEOSX );
+    std::string lifoPrefix = GEOS_FMT( "lifo/rank_{:05}/pdt2_shot{:06}", rank, m_shotIndex );
+    m_lifo = std::unique_ptr< lifoStorage< real32 > >( new lifoStorage< real32 >( lifoPrefix, p_dt2, m_lifoOnDevice, m_lifoOnHost, m_lifoSize ) );
+
+  } );
+}
+
+
```

Comment:
Same as initializePreSubGroups take a look to postProcessInput inside the AcousticWaveEquationSEM solver it should be the exact same thing